File: back-end/Entidades/Vendedor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace back_end.Entidades
{
    public class Vendedor
    {
        public int id { get; set; }
        public string nombre { get; set; }
        public string apellido { get; set; }
        public string documento { get; set; }
        public int ciudad { get; set; }
    }
}

File: front-end/src/app/vendedores/vendedor.ts
import { CiudadDto } from "../ciudades/ciudad";

export interface VendedorDto{
    id: number;
    nombre: string;
    apellido: string;
    documento: string;
    ciudad: CiudadDto[];
}

export interface CreacionVendedorDto{
    nombre: string;
    apellido: string;
    documento: string;
    ciudadId: number;
}